lms/djangoapps/mobile_api/social_facebook/courses/tests.py

```diff
@@ -0,0 +1,267 @@
+"""
+Tests for Courses
+"""
+
+from django.core.urlresolvers import reverse
+from rest_framework.test import APITestCase
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from courseware.tests.factories import UserFactory
+from xmodule.modulestore.tests.factories import CourseFactory
+from student.models import CourseEnrollment
+from student.views import login_oauth_token
+from social.apps.django_app.default.models import UserSocialAuth
+import httpretty
+import json
+from openedx.core.djangoapps.user_api.api.profile import preference_info, update_preferences
+
+
+class TestGroups(ModuleStoreTestCase, APITestCase):
+    """
+    Tests for /api/mobile/v0.5/courses/...
+    """
+
+    USERNAME_1 = "Daniel Eidan"
+    EMAIL_1 = "daniel@ebnotions.com"
+    PASSWORD_1 = "edx"
+    FB_ID_1 = "10155110991745300"
+
+    USERNAME_2 = "Marc Ashman"
+    EMAIL_2 = "marc@ebnotions.com"
```

Comment:
Sorry my comment is misleading.
The FB id's don't need to be real now that the tests have been decoupled
from FB.
The IDs need be real when this API endpoint is invoked by the Android
client because it then continues and loads the avatars directly from FB.

On Tue, Feb 10, 2015 at 3:40 PM, Nimisha Asthagiri <notifications@github.com
> wrote:

> In lms/djangoapps/mobile_api/social_facebook/courses/tests.py
> <https://github.com/edx/edx-platform/pull/6788#discussion_r24448382>:
>
> > +import json
> > +from openedx.core.djangoapps.user_api.api.profile import preference_info, update_preferences
> > +
> > +
> > +class TestGroups(ModuleStoreTestCase, APITestCase):
> > +    """
> > +    Tests for /api/mobile/v0.5/courses/...
> > +    """
> > +
> > +    USERNAME_1 = "Daniel Eidan"
> > +    EMAIL_1 = "daniel@ebnotions.com"
> > +    PASSWORD_1 = "edx"
> > +    FB_ID_1 = "10155110991745300"
> > +
> > +    USERNAME_2 = "Marc Ashman"
> > +    EMAIL_2 = "marc@ebnotions.com"
>
>  The issue is that these need to be real FB id's so that the avatars can
> be loaded properly
>
> Hmm... does that mean the unit tests are calling the FB servers to get the
> avatars? Or am I misunderstanding your comment?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/6788/files#r24448382>.
>



-- 
*Daniel Eidan | Technical Lead*

*email : *daniel@bnotions.com
*phone:* 416 858 9455

*BNOTIONS*
106 Front Street East, Suite 200
Toronto, Ontario
CANADA M5A 1E1
416 924 2784
www.bnotions.com


Love BNOTIONS <http://www.facebook.com/bnotions> on Facebook!

This email is confidential, please treat it as such.